basecoin/modules/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "basecoin-modules"
+version     = { workspace = true }
+edition     = { workspace = true }
+license     = { workspace = true }
+readme      = { workspace = true }
+repository  = { workspace = true }
+authors     = { workspace = true }
+description = """
+        Contains simple implementations of various modules for the Basecoin application.
+"""
+
+[dependencies]
+# external dependencies
+base64             = { workspace = true }
+cosmrs             = "0.15.0"
```

Comment:
```suggestion
cosmrs             = { version = "0.15.0" }
```

Reply:
should we make this workspace dependency?

Reply:
[ed23af3](https://github.com/informalsystems/basecoin-rs/pull/156/commits/ed23af315bc0755b0de03f466ae89a9f5b6d6a31)